Sunrise/ViewModels/ProductViewModel.swift

```diff
@@ -56,42 +56,24 @@ class ProductViewModel {
             guard let price = self?.priceForSize(size, variants: allVariants), value = price.value else { return "" }
 
             if let discounted = price.discounted?.value {
-                return self?.formatPriceValue(discounted) ?? ""
+                return "\(discounted)"
             } else {
-                return self?.formatPriceValue(value) ?? ""
+                return "\(value)"
             }
         }
 
         oldPrice <~ size.producer.map { [weak self] size in
             guard let price = self?.priceForSize(size, variants: allVariants), value = price.value,
                     _ = price.discounted?.value else { return "" }
 
-            return self?.formatPriceValue(value) ?? ""
+            return "\(value)"
         }
     }
 
-    private func priceForSize(size: String, variants: [ProductVariant]) -> Price? {
-        return variants.filter({ $0.attributes?.filter({ $0.name == "size" }).first?.value as? String == size }).first?
-                        .prices?.filter({ price in
-                            // Always pick the price without channel, customerGroup, country and validUntil/validFrom
-                            if price.channel == nil && price.customerGroup == nil && price.country == nil
-                                    && price.validFrom == nil && price.validUntil == nil {
-                                return true
-                            }
-                            return false
-                        }).first
-    }
+    // MARK: Internal Helpers
 
-    private func formatPriceValue(value: Money) -> String? {
-        if let centAmount = value.centAmount, currencyCode = value.currencyCode,
-                currencySymbol = NSLocale(localeIdentifier: currencyCode).displayNameForKey(NSLocaleCurrencySymbol, value: currencyCode) {
-            let currencyFormatter = NSNumberFormatter()
-            currencyFormatter.numberStyle = .CurrencyStyle
-            currencyFormatter.currencySymbol = currencySymbol
-            currencyFormatter.locale = NSLocale.currentLocale()
-            return currencyFormatter.stringFromNumber(centAmount / 100)
-        }
-        return nil
+    private func priceForSize(size: String, variants: [ProductVariant]) -> Price? {
+        return variants.filter({ $0.attributes?.filter({ $0.name == "size" }).first?.value as? String == size }).first?.independentPrice
```

Comment:
The `"size"` string is used more often, maybe it makes sense to move it to a string constant or enum here, e.g. `AttributeSize` or `Attribute.Size`?